cmd/integration/integration.go

```diff
@@ -196,7 +196,7 @@ func startComponents(firstManifestURL, secondManifestURL string) (string, string
 		Run(3, wait.NeverStop)
 
 	// TODO: Write an integration test for the replication controllers watch.
-	go replicationcontroller.NewReplicationManager(clientset, controller.NoResyncPeriodFunc, replicationcontroller.BurstReplicas).
+	go replicationcontroller.NewReplicationManager(clientset, controller.NoResyncPeriodFunc, replicationcontroller.BurstReplicas, 0).
```

Comment:
I think we shouldn't pass 0, because we want to have integration tests to be as similar to production setup as possible.
So let's set it to 4096 which is default in production setup.